### ISSUE TITLE
When upgaring C# project -> add "EnableDynamicLoading" to .csproj

### ISSUE THREAD

**Issue by Xyotic** [godotengine/godot#66725]
### Godot version

Godot4 Beta2 net6

### System information

Windows10, Vulkan, RTX2080

### Issue description

When upgrading an existing C# project to Godot4,
"<EnableDynamicLoading>true</EnableDynamicLoading>" wont get added automatically, which seems to be prerequisite to run c# scripts in general. Else Godot wont be able to find the classes.
This should be done automatically i think.

### Steps to reproduce

Have a C# project without "<EnableDynamicLoading>true</EnableDynamicLoading>" enabled in .csproj.

**Comment by raulsntos** [comment]
I don't think we support opening Godot 3 C# projects transparently in Godot 4. You'll have to let Godot recreate the `.csproj` or fix it manually. To fix the `.csproj` you'll need to change the TFM to `net6.0` and add `<EnableDynamicLoading>true</EnableDynamicLoading>`.
